frontend/website-redesign/src/components/Footer.re

```diff
@@ -0,0 +1,465 @@
+      alignContent(`spaceBetween),
+      media(Theme.MediaQuery.desktop, [marginRight(`rem(10.6))]),
+    ]);
+  let emailInputSection =
+    style([media(Theme.MediaQuery.desktop, [marginTop(`rem(10.5))])]);
+  let logo =
+    style([
+      height(`rem(3.1)),
+      width(`rem(11.)),
+      marginBottom(`rem(4.)),
+    ]);
+  let label = merge([Theme.Type.h4, style([color(white)])]);
+  let connectLabel =
+    merge([Theme.Type.h4, style([color(white), marginTop(`rem(2.12))])]);
+  let paragraph = merge([Theme.Type.paragraph, style([color(white)])]);
+  let emailSubtext =
+    merge([
+      Theme.Type.paragraph,
+      style([color(white), marginTop(`zero), marginBottom(`rem(1.))]),
+    ]);
+};
+
+module SocialIcons = {
+  module Styles = {
+    open Css;
+    let iconsRow =
+      style([
+        display(`flex),
+        flexDirection(`row),
+        justifyContent(`spaceBetween),
+        alignContent(`center),
+        width(`rem(14.)),
+        height(`rem(2.)),
+        color(white),
+      ]);
+  };
+  [@react.component]
+  let make = () => {
+    <div className=Styles.iconsRow>
+      <Icon kind=Icon.Discord size=2. />
+      <Icon kind=Icon.Twitter size=2. />
+      <Icon kind=Icon.Facebook size=2. />
+      <Icon kind=Icon.Telegram size=2. />
+      <Icon kind=Icon.WeChat size=2. />
+    </div>;
+  };
+};
+
+module FooterLinks = {
+  module Styles = {
+    open Css;
+    let linksGrid =
+      style([
+        display(`grid),
+        gridTemplateColumns([
+          `repeat((`num(2), `minmax((`rem(11.), `rem(11.5))))),
+        ]),
+        gridColumnGap(`rem(0.5)),
+        gridRowGap(`rem(3.1)),
+        marginTop(`rem(4.)),
+        media(
+          Theme.MediaQuery.tablet,
+          [
+            marginTop(`rem(5.8)),
+            gridRowGap(`rem(3.)),
+            gridColumnGap(`rem(5.)),
+            gridTemplateColumns([
+              `repeat((`num(3), `minmax((`rem(11.), `rem(11.5))))),
+            ]),
+          ],
+        ),
+        media(
+          Theme.MediaQuery.desktop,
+          [marginTop(`rem(0.)), width(`rem(36.5)), height(`rem(30.))],
+        ),
+      ]);
+    let linksGroup =
+      style([
+        display(`flex),
+        flexDirection(`column),
+        alignContent(`flexStart),
+        flexWrap(`wrap),
+      ]);
+    let linksHeader =
+      merge([
+        Theme.Type.footerHeaderLink,
+        style([
+          marginTop(`zero),
+          marginBottom(`zero),
+          color(white),
+          opacity(0.4),
+        ]),
+      ]);
+
+    let linkStyle =
+      merge([
+        Theme.Type.sidebarLink,
+        style([
+          marginTop(`rem(0.5)),
+          color(white),
+          textDecoration(`none),
+        ]),
+      ]);
+  };
+  [@react.component]
+  let make = () => {
+    <div className=Styles.linksGrid>
+      <div className=Styles.linksGroup>
+        <h4 className=Styles.linksHeader> {React.string("Get Started")} </h4>
+        <Next.Link href="">
```

Comment:
does it make sense to add a TODO comment at the top of this component to make sure we add these links? one pro is that it'd make it easier for searching later.